app/src/main/java/com/brainwallet/di/Module.kt

```diff
@@ -94,28 +98,6 @@ private fun provideOkHttpClient(): OkHttpClient = OkHttpClient.Builder()
             .addHeader("Accept-Language", "en")
         chain.proceed(requestBuilder.build())
     }
-    .addInterceptor { chain ->
-        val request = chain.request()
-        runCatching {
-            chain.proceed(request).use { response ->
-                if (response.isSuccessful.not()) {
-                    throw HttpException(
-                        retrofit2.Response.error<Any>(
-                            response.code,
-                            response.body ?: response.peekBody(Long.MAX_VALUE)
-                        )
-                    )
-                }
-                response
-            }
-        }.getOrElse {
-            //retry using dev host
-            val newRequest = request.newBuilder()
```

Comment:
@kcw-grunt can we remove this retry to dev host? since the dev host didn't have latest update like production host?
cc @josikie

Reply:
Yes please @andhikayuana . The cloud architecture will be routing all calls to api.grunt.ltd only and we will scale in AWS.